dev/io.openliberty.mail.2.1.internal_fat/build.gradle

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+/* Define greenmail dependencies */
+dependencies {
+  requiredLibs project(':com.ibm.ws.org.slf4j.api'),
+               project(':com.ibm.ws.org.slf4j.jdk14'),
+               project(':io.openliberty.org.eclipse.angus.mail'),
+               project(':com.ibm.websphere.javaee.jaxb.2.2'),
```

Comment:
Please can you double check this version of jaxb? If this is a JEE10 FAT then shouldn't this be `io.openliberty.jakarta.xmlBinding.4.0`

Reply:
For mail.2.0.internal_fat, greenmail server used in tests has jaxb.2.2 dependencies. Otherwise, we are not using JAX-B anywhere in our code. Having said that for mail.2.1.internal_fat we were not using this mail server. I removed build.grade file completely.